op-chain-ops/genesis/testdata/test-deploy-int-nets.json

```diff
@@ -0,0 +1,45 @@
+{
+  "l1StartingBlockTag": "earliest",
+  "l1ChainID": 900,
+  "l2ChainID": 901,
+  "l2BlockTime": 2,
+
+  "maxSequencerDrift": 100,
+  "sequencerWindowSize": 4,
```

Comment:
Any way that we can not introduce this by using a combo of 1/0 and local/remote assertions against existing files?